nipype/algorithms/confounds.py

```diff
@@ -236,6 +252,14 @@ class FramewiseDisplacementInputSpec(BaseInterfaceInputSpec):
         "NIPY",
         desc="Source of movement parameters",
         mandatory=True)
+    metric = traits.Enum(
+        'L1',
+        'riemannian',
+        usedefault=True,
```

Comment:
Can you re-add `mandatory=True`? Without it, it's possible for users to un-set the attribute by using `metric=Undefined`, which should be an error.